website/docs/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-upgrading-dbt-versions.md

```diff
@@ -0,0 +1,192 @@
+---
+title: "Upgrading dbt versions in Cloud"
+id: "cloud-upgrading-dbt-versions"
+---
+
+This Q&A guide should help you figure out what changes you might need to make to successfully upgrade your version of dbt Core in dbt Cloud. As a reminder, we recommend everyone upgrade to the most recent version of dbt, as we will not support running all versions of dbt in Cloud indefinitely. We document which versions of dbt Core we support [here](docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-choosing-a-dbt-version#supported-versions). 
+
+There aren't many breaking changes between versions, and it may be the case that you don't need to change any code to upgrade to a newer version of dbt in dbt Cloud. You may only need to change the settings in your environment or job to call a more recent version of dbt - directions to do so can be found [here](/docs/dbt-cloud/cloud-configuring-dbt-cloud/cloud-choosing-a-dbt-version.md).
+
+
+### How to run the latest version of dbt in Cloud
+
+#### Changes between minor versions of dbt that will affect your project
+
+Below we try to help you answer the question of whether a known breaking change between minor versions of dbt will affect your project. If you answer "yes" to any of the questions below, we recommend that you read the migration guides that we've put together for every dbt minor version release.
+
+:::info An Important Note on Packages
+
+If you use any packages from [dbt Hub]([https://hub.getdbt.com/](https://hub.getdbt.com/)), make sure you also upgrade to a version of the package that supports the dbt version you intend to upgrade to. You can see which dbt versions a package supports by checking on the `require-dbt-version:` in the package's dbt_project.yml file on GitHub.
+
+As an example, dbt-utils version 0.7.1 supports dbt v0.20 and v0.21, as described in its [dbt_project.yml](https://github.com/dbt-labs/dbt-utils/blob/0.7.1/dbt_project.yml).
+
+After you've changed the package version in your packages.yml file, be sure to run `dbt deps` in the IDE to install the updated version.
+
+:::
+
+
+<details>
+<summary>  Upgrading to v0.20.latest from v0.19 </summary>
+<br></br>
+- Does your project define any custom schema tests?
```

Comment:
I think these are formatting weird in the preview! 
![image](https://user-images.githubusercontent.com/60658459/134073404-7a0a2d13-f0ca-407a-abd3-cf7636973949.png)

'Upgrade to v0.19.latest from v0.18' and 'Upgrading to v0.17.latest from v0.16' both look good in the preview, but all others don't seem to be formatting correctly!